.github/workflows/maven_test.yml

```
@@ -41,6 +41,11 @@ on:
         required: false
         type: string
         default: ubuntu-latest
+      architecture:
```

```suggestion
      arch:
```
as we use abbr for `os` and `envs`

---

done